package.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "leblad-sdk",
```

Comment:
usually npm packages names are simple, plus currently the name `leblad` is available, i think it's better to keep it `leblad` what u think?

Reply:
That's what I did 😅  see [L2](https://github.com/dzcode-io/leblad/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R2) addition: 
```
+  "name": "leblad",
```